src/vivarium_gates_nutrition_optimization/model_specifications/model_spec.yaml

```diff
@@ -24,8 +24,8 @@ components:
             - BirthRecorder()
             - MaternalMortalityObserver()
             - DisabilityObserver()
-            - AnemiaObserver()
-            - MaternalBMIObserver()
+#            - AnemiaObserver()
+#            - MaternalBMIObserver()
```

Comment:
We don't need these outputs for the emulator, but I just want to confirm that commenting these out won't prevent this information from being observed in the birth recorder for use in initializing the child simulation.

Reply:
Otherwise I don't have any concerns about the observers that were commented out!

Reply:
This will work as expected